dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeAPIImpl.java

```diff
@@ -631,6 +631,20 @@ private void addEmptyContainers(final IHTMLPage page,
         }
     }
 
+    private boolean isContains(
```

Comment:
Please add some Javadoc here, the method's name is not clear, or change it

Reply:
rename to contains